### ISSUE TITLE
Should PuTTY be an option for Windows users on the course?

### ISSUE THREAD

**Issue by mattagape** [hpc-carpentry/hpc-shell#23]
As was mentioned in issue #22, there are some problems with PuTTY.

Specifically, although it is a great tool for Windows users to access a remote Unix server, it does not have the ability to act in a Unix-like way on a user's local Windows PC.

This issue is to act as a placeholder to discuss this. Should we recommend PuTTY or something(s) else?

**Comment by tkphd** [comment]
I think we should recommend using Git for Windows, since it's a full-featured terminal already used in the Software Carpentries workshops. PuTTY is an excellent small utility, but bash provides users/learners with a lot more flexibility.

Related to https://github.com/hpc-carpentry/hpc-intro/issues/20

**Comment by psteinb** [comment]
Agreed, I think keeping the ties to other carpentry material is a good idea. Let’s stick with the recommendations of shell-novice as they don’t compromise our material in any way.

**Comment by mattagape** [comment]
There is some support [on Windows](https://www.microsoft.com/en-us/p/ubuntu/9nblggh4msv6?activetab=pivot:overviewtab) for Linux, e.g. 

I'm not sure if this is just for Windows 10 though. Similarly there are [more flavours](https://www.microsoft.com/en-gb/search?q=linux) in the Windows App Store, but again I'm not sure which versions of Windows can use them.
